kustomize/apps/profiles/base/namespace-labels.yaml

```diff
@@ -0,0 +1,4 @@
+katib.kubeflow.org/metrics-collector-injection: "enabled"
```

Comment:
need to add the istio label here `istio-injection: "enabled"`

Reply:
It seems to be merged already on dev:

https://github.com/StatCan/aaw-kubeflow-manifests/blob/aaw-dev-cc-00/kustomize/apps/profiles/base/namespace-labels.yaml

Reply:
But I am missing one of the labels. For instance, my PR for prod has the following labels:

```
istio-injection: "enabled"
katib.kubeflow.org/metrics-collector-injection: "enabled"
serving.kubeflow.org/inferenceservice: "enabled"
pipelines.kubeflow.org/enabled: "true"
app.kubernetes.io/part-of: "kubeflow-profile"
```

And dev has the following labels right now:

```
istio-injection: "enabled"
katib.kubeflow.org/metrics-collector-injection: "enabled"
serving.kubeflow.org/inferenceservice: "enabled"
pipelines.kubeflow.org/enabled: "true"
```

Should I add `app.kubernetes.io/part-of: "kubeflow-profile"` to dev?